### PR TITLE
feat: (Series|DataFrame).plot

### PR DESCRIPTION
**Pull request by chelsea-lin**
Fixes internal bug: b/322178330 🦕

### CODE REVIEW

File: third_party/bigframes_vendored/pandas/plotting/_core.py [hunk 1]
```diff
@@ -6,8 +6,166 @@
...
+
+        - ``False`` : No subplots will be used
+        - ``True`` : Make separate subplots for each column.
+        - sequence of iterables of column labels: Create a subplot for each
+          group of columns. For example `[('a', 'c'), ('b', 'd')]` will
+          create 2 subplots: one with columns 'a' and 'c', and one
+          with columns 'b' and 'd'. Remaining columns that aren't specified
+          will be plotted in additional subplots (one per column).
+
+          .. versionadded:: 1.5.0
```
Comment (junyazhang) [inline third_party/bigframes_vendored/pandas/plotting/_core.py:34]: Given that we are extending pandas, shall we keep this note here?

File: third_party/bigframes_vendored/pandas/plotting/_core.py [hunk 2]
```diff
@@ -6,8 +6,166 @@
...
+    stacked : bool, default False in line and bar plots, and True in area plot
+        If True, create stacked plot.
+    secondary_y : bool or sequence, default False
+        Whether to plot on the secondary y-axis if a list/tuple, which
+        columns to plot on secondary y-axis.
+    mark_right : bool, default True
+        When using a secondary_y axis, automatically mark the column
+        labels with "(right)" in the legend.
+    include_bool : bool, default is False
+        If True, boolean values can be plotted.
```
Comment (junyazhang) [inline third_party/bigframes_vendored/pandas/plotting/_core.py:134]: Do we support all these parameters?

File: third_party/bigframes_vendored/pandas/plotting/_core.py
Comment (junyazhang) [inline third_party/bigframes_vendored/pandas/plotting/_core.py:1]: Can you paste the staged doc link here?

File: third_party/bigframes_vendored/pandas/plotting/_core.py (hunk 1 above)
Comment (chelsea-lin) [reply third_party/bigframes_vendored/pandas/plotting/_core.py:34]: Good catch. Removed!

File: third_party/bigframes_vendored/pandas/plotting/_core.py (hunk 2 above)
Comment (chelsea-lin) [reply third_party/bigframes_vendored/pandas/plotting/_core.py:134]: The implementation of existing plotting methods are converting BF DataFrames and calling `pandas.DataFrame.plot()`. Hence, all arguments listed at [pandas.DataFrame.plot()](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.plot.html) should also support here technically. Though I didn't have chance to test all these arguments yet. 

Maybe I would just comment them as `Options to pass to pandas.DataFrame.plot method.` for now?

**Review by junyazhang — APPROVED** [review]

**Pull request merged**